bindings/pyroot_experimental/PyROOT/src/CPPInstancePyz.cxx

```diff
@@ -0,0 +1,123 @@
+// Author: Massimiliano Galli CERN  07/2019
+// Original PyROOT code by Wim Lavrijsen, LBL
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+// Bindings
+#include <iostream>
+#include "CPyCppyy.h"
+#include "PyROOTPythonize.h"
+#include "CPPInstance.h"
+#include "ProxyWrappers.h"
+#include "Converters.h"
+#include "Utility.h"
+#include "PyzCppHelpers.hxx"
+#include "TBufferFile.h"
+#include "CustomPyTypes.h"
+
+using namespace CPyCppyy;
+
+namespace PyROOT {
+    extern PyObject* gRootModule;
+}
+
+/// This method is a helper for (un)pickling of ObjectProxy instances.
```

Comment:
This one also belongs to the PyROOT extension module API, and therefore needs Doxygen documentation too.
In the description of the function, I would explain why we need to include it in the extension module API: otherwise it is not callable from Python, and it will be Python who will call it when trying to expand a serialized object.